### ISSUE TITLE
Packaging interfaces

### ISSUE THREAD

**Issue by argenos** [HBRS-MAAS/project#31]
Two pull requests are needed:

- [ ] docs/messages/loading-bay.md containing the input and output messages
- [ ]    the loading bay agent, which inherits from the base class

Related to #47

**Comment by argenos** [comment]
This stage the loading agent should group the boxes into orders, and make a request to @HBRS-MAAS/commitment_issues for the orders that they delivered. 

@HBRS-MAAS/commitment_issues will figure out how to best split them in the trucks, and when to pick them up, according to all the products they have to deliver (eventually from multiple bakeries) and their planned routes and the delivery deadlines.

**Comment by argenos** [comment]
Since you need to interface with the packaging stage, please review this @HBRS-MAAS/commitment_issues

**Comment by Sushant-Chavan** [comment]
@HBRS-MAAS/team-trk  and @argenos  Since we did not receive any output message format for the loading bay, we would like to propose the below format for the same:

```
{
  "OrderID": "Order-123",
  "Boxes": [
    {
      "BoxID": "1",
      "ProductType": "Bread",
      "Quantity": 12
    },
    {
      "BoxID": "2",
      "ProductType": "Bread",
      "Quantity": 6
    },
    {
      "BoxID": "3",
      "ProductType": "Donut",
      "Quantity": 5
    }
  ]
}
```

**Comment by argenos** [comment]
@HBRS-MAAS/team-trk You need to create a pull request with your messages **today**, as they are needed by the other teams who are working on this. This was due last Wednesday.

**Comment by argenos** [comment]
Merged in #67
